toolbox/bulk_processing/validators.py

```diff
@@ -284,3 +285,11 @@ def validate(value: str, **_kwargs):
             raise Invalid(f'Value {value} must be either a whole number between 0 and 99 or delete keyword "-----"')
 
     return validate
+
+
+def is_case_id_a_valid_uuid(case_id):
```

Comment:
Small point about naming convention, I would prefer a name like `fail_validation_if_invalid_uuid` to be clear it'll raise an exception, I would typically expect `is_...` methods to return a True/False rather than raise.